tests/test_servers.py

```
@@ -0,0 +1,359 @@
+            if s["min_price_ondemand"] is not None:
+                assert s["min_price"] == s["min_price_ondemand"]
+
+    def test_monthly(self):
+        data, _ = get_servers(best_price_allocation="MONTHLY", limit=10)
+        for s in data:
+            if s["min_price_ondemand_monthly"] is not None:
+                assert s["min_price"] == s["min_price_ondemand_monthly"]
+
+
+# ---------------------------------------------------------------------------
+# Monthly traffic – price_breakdown
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)